payment_signal/models/inherit_res_company.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    default_signal = fields.Integer('Percent Signal in S.O.',
+                                    default=20,
+                                    store=True)
+
+    @api.constrains('default_signal')
```

Comment:
use api.multi here and ensure one.

Reply:
I'm sorry I do not understand it. If I do not put a "constrain", you can save the field with a negative value or greater than 100. And I do not want that.

I don't understand "ensure one".

Thank you very much.